src/EPR.RegulatorService.Frontend.Web/Views/Registrations/RegistrationDetails.cshtml

```diff
@@ -8,7 +8,7 @@
 
 @model RegistrationDetailsViewModel
 @{
-    ViewData["Title"] = Localizer["Title.ManageRegistrationDataSubmissions"];
+    ViewData["Title"] = Localizer["Title.RegistrationsOrgTitle"];
```

Comment:
Can you correct the names so it is consistent, if we're going to start using the . notation then lets have X.Title consistently instead of Title.X, One is "OrganisationDetailsFileDownloadFailed.Title" and this is "Title.RegistrationsOrgTitle", Title appears twice too.

Reply:
Hi Simon, that's a really good point. As we discussed over the Teams Chat I have now updated all the Titles.